_posts/2020-09-15-solidity-underhanded-contest.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+published: true
+title: 'The Solidity Underhanded Contest is back!'
+date: '2020-09-18'
+author: Franziska Heintel
+category: Announcements
+---
+
+We're excited to share that the [Solidity Underhanded Contest](https://underhanded.solidity.rocks/) is finally back!
+
+Inspired by the [Underhanded C Contest](http://www.underhanded-c.org/) and the first [Underhanded Solidity Contest](https://u.solidity.cc/), organized in 2017 by Nick Johnson, we decided it is time for a much needed revival.
+
+![Solidity Underhanded Contest](/img/2020/09/Underhanded_Solidity.gif)
+
+The goal of this contest is to write innocent-looking Solidity code, which pretends to clear and straightforward, but actually contains malicious behavior or backdoors.
+
+By hosting such a contest we aim to:
+
++ Raise awareness about smart contract security.
++ Uncover language design faults and backdoors.
++ Highlight anti-patterns in smart contact development.
++ Establish new best practices for secure smart contract development.
+
+## The Theme 🚪🔑
+
+**This year's theme is upgradability of contracts, or, more specifically, upgrade mechanisms.**
+
+Since a smart contract in which a single account is authorized to arbitrarily change its implementation defeats the idea of decentralization, submissions need to contain a mechanism where code can be upgraded but users are (seemingly) still in control. 
```

Comment:
The first and the second part of this sentence do not really match up - it sounds like a connecting part is missing.

Reply:
You are totally right. I tried to connect them. What do you think?